articles/openshift/oshift-how-add-domains-proxy-whitelist.md

```diff
@@ -0,0 +1,130 @@
+---
+title: How to permit outbound access to internet hosts in deployments with government community network connectivity | UKCloud Ltd
+description: Outlines methods to securely permit access to internet hosts in v3.11 OpenShift clusters deployed with government community network connectivity.
+services: openshift
+author: Ben Bacon
+toc_rootlink: How To
+toc_sub1: 
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Permit outbound internet access in deployments with government community network connectivity
+toc_fullpath: How To/oshift-how-add-domains-proxy-whitelist.md
+toc_mdlink: oshift-how-add-domains-proxy-whitelist.md
+---
+
+# How to permit outbound access to internet hosts in deployments with government community network connectivity
+
+## Overview
```

Comment:
We should add something in the overview to point out that this shouldn't be done lightly and should be included in any security documentation that is submitted to relevant authorities for accreditation or something along those lines anyway.